### PR TITLE
Filter menu

### PR DESCRIPTION
**Pull request by henderson907**
- Added a filter feature for venue types
- Added styling for the checkboxes so they appear as buttons
- Added styling to buttons to create better front-end appearance
- Fixed error with filter algorithm so it now takes into account all filters rather than the last one
- Added comments to venues controller as the code was getting complex

image
image

### CODE REVIEW

File: app/assets/stylesheets/pages/_venues.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+.filter-form-button {
+  padding-top: 15px;
+  padding-bottom: 15px;
+  font-size: 25px;
+  margin: 15px;
+}
+
+.filter-option-btn {
```
[the remark this answers is not in the document]
Comment (henderson907) [reply app/assets/stylesheets/pages/_venues.scss:22]: fixed

File: app/assets/stylesheets/pages/_venues.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+.filter-menu {
+  display: flex;
+  justify-content: space-around;
+}
+
+.filter-form {
+  display: flex;
+  flex-direction: column;
```
[the remark this answers is not in the document]
Comment (henderson907) [reply app/assets/stylesheets/pages/_venues.scss:12]: fixed

File: app/assets/stylesheets/components/_bubble.scss [hunk 3]
```diff
@@ -7,5 +7,25 @@
...
+  background-color: white;
+  border-radius: 50px;
+  color: $blue;
+  border: 1px solid $blue;
+  display: inline-block;
+  font-size: 15px;
+  font-weight: bold;
+  margin: 10px 10px 10px 0px;
+  padding: 15px 25px;
   text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```
Comment (pmunozs) [inline app/assets/stylesheets/components/_bubble.scss:23]: I would suggest removing this line for the invert-bubble. The text seems a little bit more clear when we don't have the shadow

image

File: app/assets/stylesheets/pages/_venues.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (pmunozs) [reply app/assets/stylesheets/pages/_venues.scss:12]: I'm still seeing it overflown 
image

File: app/assets/stylesheets/components/_bubble.scss (hunk 3 above)
Comment (henderson907) [reply app/assets/stylesheets/components/_bubble.scss:23]: okay, done

**Review by pmunozs — APPROVED** [review]

**Pull request merged**